packages/common/src/utils/nft.utils.ts

```diff
@@ -124,29 +127,31 @@ export class NftUtils {
   /**
    * Identifies if the metadata for a NFT needs updating and, if it does, update it.
    * @param {string} nftUid
+   * @param {number} maxRetries
```

Comment:
Missing `logger` parameter

Reply:
Fixed in 16efa8fdcf591e6e3c8b74a7e8e9e7bcb2f2d9e2